eve-api-data/src/test/java/com/eveonline/api/repo/AccountBalanceTests.java

```diff
@@ -20,10 +16,9 @@
 	@Test
 	@Sql({"/testGetAccountId34339379Balance.sql"})
 	public void testGetAccountId34339379Balance() {
-		AccountBalance balance = repo.findByAccountKeyAndCorporationId(1000, 1l);
```

Comment:
This was a hint that some type issue's would arise when the unit test was executed.